regolith/helpers/l_grantshelper.py

```diff
@@ -63,20 +63,17 @@ def construct_global_ctx(self):
     def sout(self):
         rc = self.rc
         grants = []
-        desired_date = None
         if rc.date:
             desired_date = date_parser.parse(rc.date).date()
-        elif rc.current:
+        else:
             desired_date = dt.date.today()
         for grant in self.gtx["grants"]:
-            if desired_date:
-                try:
-                    if is_current(grant, desired_date):
-                        grants.append(grant)
-                except RuntimeError:
+            try:
+                if rc.current and not is_current(grant, now=desired_date):
                     continue
-            else:
-                grants.append(grant)
+            except RuntimeError:
```

Comment:
We need to take this out of the try except block or it might not crash if the database is bad.

Reply:
I put it in a try catch because is_current can rise a RuntimeError is that grant does not have a defined behavior. I am assuming that it is possible for grants to not have start or end dates and if so I do not want the program to crash for those grants